ios/Exponent/Kernel/AppLoader/EXAppLoader.m

```diff
@@ -37,6 +37,7 @@ @interface EXAppLoader ()
 
 @property (nonatomic, assign) BOOL hasFinished;
 @property (nonatomic, assign) BOOL shouldUseCacheOnly;
+@property (nonatomic, assign) BOOL isUpToDate;
```

Comment:
Shouldn't need this in the internal interface since it's defined in the public one I think